### PR TITLE
Add usage examples with JSON --input

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Added examples in --help on how to pass --input for 
- `temporal workflow start`
- `temporal workflow execute`
- `temporal workflow signal`
- `temporal schedule create`

## Why?

resolve https://github.com/temporalio/cli/issues/208

## Checklist

1. Closes 

2. How was this tested:

Executed each example manually

3. Any docs updates needed?

### CODE REVIEW

File: common/defs-cmds.go [hunk 1]
```diff
@@ -142,15 +142,24 @@ Workflow commands use this syntax:
...
+		--input='{"Input": "As-JSON"}'
+` + "`" + `` + "`" + `` + "`" + `
 
 Use the command options below to change the information returned by this command.`
 
 const ExecuteWorkflowUsageText = `The ` + "`" + `temporal workflow execute` + "`" + ` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and prints its progress.
 The command completes when the Workflow Execution completes.
 
-[Workflows](/concepts/what-is-a-workflow) are executed with the following syntax:
-` + "`" + `temporal workflow execute --workflow-id=meaningful-business-id --type=MyWorkflow --task-queue=MyTaskQueue` + "`" + `
```
Comment (bergundy) [inline common/defs-cmds.go:153]: Leave the workflow ID, we encourage users to set it.

**Review by bergundy — APPROVED** [review]
LGTM, left one comment.
There are two stylistic choices that I don't love:
- The use of `=` for option values, I prefer a single space
- Breaking up shortish lines into multiple lines

**Comment by feedmeapples** [comment]
> * Breaking up shortish lines into multiple lines

agreed about shortish, though even the execute one looks pretty long
image

When it's split into multiple lines it also stands out better imo in the help output
image

**Comment by feedmeapples** [comment]
> * The use of `=` for option values, I prefer a single space

Right, updated

**Pull request merged**